src/config/cli.rs

```diff
@@ -66,12 +66,28 @@ pub struct Cli {
     /// Enable default settings for debugging contracts.
     pub debug_mode: bool,
 
+    #[arg(long, default_value = "true", default_missing_value = "true", num_args(0..=1), help_heading = "Debugging Options")]
+    /// If true, prints node config on startup.
+    pub show_node_config: Option<bool>,
+
+    #[arg(long, default_value = "true", default_missing_value = "true", num_args(0..=1), help_heading = "Debugging Options")]
+    /// If true, prints calls and transactions summary.
+    pub show_calls_summary: Option<bool>,
```

Comment:
Perhaps rename this as its too close to `show_calls` while this flag is for transaction summary / outcome

Reply:
Thought about that, but I would better rename show_calls as it is not clear. The flag I added is for both eth_call and transactions.

Reply:
Renamed to show_tx_summary